openedx/core/djangoapps/schedules/resolvers.py

```diff
@@ -357,16 +361,24 @@ def schedules_for_bin(self):
             except CourseUpdateDoesNotExist:
                 continue
 
-            course_id_str = str(enrollment.course_id)
             template_context.update({
                 'course_name': schedule.enrollment.course.display_name,
-                'course_url': reverse('course_root', args=[course_id_str]),
+
+                # For us to be able to track clicks in the email, this URL needs to point to a landing page that does
+                # not result in a redirect so that the GA snippet can register the UTM parameters.
```

Comment:
Why not put this comment once in the helper function instead of in each of the 3 calls?  Future maintenance/edits of the comment could lead to confusion.

Reply:
I'm afraid of someone just looking at the context building stuff and not actually reading that function... I'm OK with putting it in the function though, I'll make that change now.

Reply:
Suggestion, if wanted: In that case, you can rename the function to something like `_get_trackable_course_url` - and then interested parties can follow-through to understand what makes it trackable.